docs/platform/changelog.md

```diff
@@ -4,8 +4,24 @@ description: What's new and improved in Grid.
 import Note from "@site/src/components/Note";
 
 # :partying_face: Releases
+Upgrade your CLI with `pip install lightning-grid --upgrade`
+## :wrench: February 28, 2022
 
-**Archived release notes can be found on** [**this page**](https://github.com/gridai/gridai/discussions) **in Github.**&#x20;
+**CLI version: 0.8.0**
+
+Spring cleaning came early. This release features a lot of backend magic that improves overall stability and UX with Grid. 
+
+Upgrade your CLI to `v0.8.0` with `pip install lightning-grid --upgrade`  
+
+Find us in our [Slack Community](http://gridai-community.slack.com) to say hi and/or to express your thoughts/questions.
+
+### Notable Fixes and Enhancements
```

Comment:
Can we add some of the notable datastore improvements here? This is something I think the community would really like to see. I believe the notable ones are

1.  Improved upload speeds
2. Fix of the Aorpc error

Reply:
Would be great to have you @rlizzo comment on the notable datastore changes.

Reply:
Thanks, femi!